src/status_im/contexts/communities/overview/view.cljs

```diff
@@ -210,18 +209,21 @@
 (defn- unusable-area-height
   []
   (+ (safe-area/get-top)
+     12
      32
      11 ;;top page buttons & button's padding
      (safe-area/get-bottom)))
 
 (defn- calc-scrollable-content
   [scrollable-height]
   (- (+ scrollable-height (unusable-area-height))
-     (:height (rn/get-window))))
+     (:height (safe-area/get-window))))
 
 (defn- calc-listing-height
   []
-  (- (:height (rn/get-window)) (unusable-area-height)))
+  (- (:height (safe-area/get-window))
+     (unusable-area-height)
+     (- (safe-area/get-bottom))))
```

Comment:
This is a nice example of how I think we can avoid referring to the special numbers like `12` or etc.